src/java.base/share/classes/java/util/concurrent/StructuredTaskScope.java

```diff
@@ -1197,7 +1197,7 @@ public static final class ShutdownOnFailure extends StructuredTaskScope<Object>
          *
          * <p> Construction captures the current thread's {@linkplain ScopedValue scoped
          * value} bindings for inheritance by threads started in the task scope. The
-         * <a href="#TreeStructure">Tree Structure</a> section in the class description
+         * {@linkplain StructuredTaskScope##TreeStructure Tree Structure} section in the class description
```

Comment:
You can drop the changes to STS if you want as we have an update coming that replaces these paragraphs/links.

Reply:
Done, thanks for the heads up!